grpc-server/src/main/java/com/netflix/conductor/grpc/server/GRPCServerProvider.java

```diff
@@ -43,15 +46,25 @@ public GRPCServerProvider(
     @Override
     public Optional<GRPCServer> get() {
         return configuration.isEnabled() ?
-                Optional.of(
-                        new GRPCServer(
-                                configuration.getPort(),
-                                healthServiceImpl,
-                                eventServiceImpl,
-                                metadataServiceImpl,
-                                taskServiceImpl,
-                                workflowServiceImpl
-                        ))
+                Optional.of(getGRPCServer())
                 : Optional.empty();
     }
+
+    private GRPCServer getGRPCServer() {
```

Comment:
**Nit:** Rename to `buildGRPCServer`.

**Suggestion:** Pass reflectionEnabled and port as arguments, or pass the configuration as an argument.  I think it improves clarity and testability.